content/paths/files__put_files_id.yml

```diff
@@ -110,6 +109,23 @@ requestBody:
                   - open
                   - company
 
+          collections:
+            type: array
+            description: |-
+              An array of collections to make this file
+              a member of. Currently
+              we only support the `favorites` collection.
+
+              To get the ID for a collection, use the
+              [List all collections][1] endpoint.
+
+              Passing an empty array `[]` or `null` will remove
+              the file from all collections.
+
+              [1]: ../advanced-files-and-folders/#get-collections
```

Comment:
I have a question about this link - do you mean https://developer.box.com/reference/get-collections/?  I couldn't find any `advanced-files-and-folders` directory in our documentation. 
If so, we usually use this convention:
`e://get-collections` - where `e` redirects to the given endpoint.